README.rst

```diff
@@ -0,0 +1,300 @@
+========================================================
+ REANA example - Search for the rare charm decays
+========================================================
+
+.. image:: https://img.shields.io/travis/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://travis-ci.org/reanahub/reana-demo-lhcb-d2pimumu
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-lhcb-d2pimumu.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-lhcb-d2pimumu/master/LICENSE
+
+
+
+About
+======
+
+This example is based on the `analysis-case-study <https://github.com/atrisovic/analysis-case-study>`_.
+
+Analysis structure
+===================
+
+Making a research data analysis reproducible basically means to provide
+"runnable recipes" addressing (1) where is the input data, (2) what software was
+used to analyse the data, (3) which computing environments were used to run the
+software and (4) which computational workflow steps were taken to run the
+analysis. This will permit to instantiate the analysis on the computational
+cloud and run the analysis to obtain (5) output results.
+
+
+1. Input data
+-------------
+The data being used is currently only available on request. It consists of a 10 GB ROOT file.
+
+2. Analysis code
+----------------
+
+You can either run the analysis step by step (explained below), or use the bash script :code:`run.sh` via:
+
+.. code-block:: console
+
+   $ bash run.sh <data_file> <tmp_phi_models> <result_plot>
+
+Step 0:
+
+`lhcbStyle.C <code/lhcbStyle.C>`_ -
+  Plot formats specific to the LHCb collaboration.
+
+
+`RooFitHeaders.h <code/RooFitHeaders.h>`_ -
+  Libraries and packages needed to perform the fitting.
+
+
+Step 1. `Optimise.C <code/Optimise.C>`_ -
```

Comment:
Can we also name all the steps instead of just using numbers?  E.g. "step 1: optimise"?